From ee9a85e95b434294e00f913fe9a5d2ce51627daf Mon Sep 17 00:00:00 2001
From: Yinbo Zhu <yinbo.zhu@nxp.com>
Date: Tue, 25 Sep 2018 15:38:58 +0800
Subject: [PATCH 305/706] arm64: dts: Add usb controller for ls1028a

Add usb controller nodes to enable usb driver.

Signed-off-by: Yinbo Zhu <yinbo.zhu@nxp.com>
(cherry picked from commit acfee82c29112acacdd1324acba8f4533a7657a1)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index 8551cdefda71..3f369e1dc133 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -302,6 +302,20 @@
 			status = "disabled";
 		};
 
+		usb0:usb3@3100000 {
+			compatible= "snps,dwc3";
+			reg= <0x0 0x3100000 0x0 0x10000>;
+			interrupts= <0 80 0x4>;
+			dr_mode= "host";
+		};
+
+		usb1:usb3@3110000 {
+			compatible= "snps,dwc3";
+			reg= <0x0 0x3110000 0x0 0x10000>;
+			interrupts= <0 81 0x4>;
+			dr_mode= "host";
+		};
+
 		i2c0: i2c@2000000 {
 			compatible = "fsl,vf610-i2c";
 			#address-cells = <1>;
-- 
2.17.1

